storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java

```diff
@@ -103,9 +102,7 @@ public String toString() {
     /**
      * Builder for {@link Query}
      */
-    public static class Builder {
-
-        Uri uri;
+    public static final class Builder {
```

Comment:
It won't work, you'll have to use custom Query class for that or modify
sources of Query class and build StorIO from sources. Yep, I want to hide
and make final everything to be sure that won't have to support inheritance
of our classes

вс, 26 апр. 2015, 19:57, Dmitrii Nikitin <notifications@github.com>:

> In
> storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java
> <https://github.com/pushtorefresh/storio/pull/302#discussion_r29111559>:
>
> > @@ -103,9 +102,7 @@ public String toString() {
> >      /**
> >       * Builder for {@link Query}
> >       */
> > -    public static class Builder {
> > -
> > -        Uri uri;
> > +    public static final class Builder {
>
> class CustomBuilder exteds Query.Builder {
> CompleteBuilder uri(java.net.URI javaUri) {
> return(javaUri.toString());
> }
> }
> and then
> storIOContentResolver()
> .get()
> .cursor()
> .withQuery(new Query.CustomBuilder()
> .uri(existJavaUri)
> .build())
> `
> Of cause, it is far-fetched example, but are you sure that it will be not
> necessary or just want to hide all except really important?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/302/files#r29111559>.
>